scrapi/harvesters/utnknoxville.py

```diff
@@ -0,0 +1,34 @@
+'''
+Harvester for the Trace: Tennessee Research and Creative Exchange for the SHARE project
+Example API call: http://trace.tennessee.edu/do/oai/?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class UtnknoxvilleHarvester(OAIHarvester):
+    short_name = 'utnknoxville'
```

Comment:
Since they call their repository "trace" should we consider that for the shortname? UT is a sorthand we've used for the University of Texas, so I was confused at first.